script.js

```diff
@@ -114,7 +114,7 @@ const renderTodoList = (todoListState, onDestroy) => {
 
   const createNewTodoItem = () => {
     if (!inputField.value) {
-      return;
+      inputField.removeEventListener('keydown',  createNewTodoItem());
```

Comment:
Let's rollback the line as we decided on the call

Reply:
OMG! Missed here) Fixed forever!)